src/tests/ftest/harness/core_files.yaml

```diff
@@ -0,0 +1,15 @@
+hosts:
+  test_servers: 2
+timeout: 120
+server_config:
+  engines_per_host: 1
+  engines:
+    0:
+      storage:
+        0:
+          class: ram
+          scm_mount: /mnt/daos
+          scm_size: 16
```

Comment:
To match the release/2.2 version of this file, https://github.com/daos-stack/daos/blob/release/2.2/src/tests/ftest/harness/advanced.yaml, this should be removed as there is not test harness code in release/2.2 to read this.